tasks/bootstrap.yml

```diff
@@ -48,7 +48,7 @@
   with_items: "{{ sb_debian_base_extra_packages }}"
 
 - include: bootstrap-firewall.yml
-  when: sb_debian_base_firewall is defined
+  when: sb_debian_base_firewall == 'yes'
```

Comment:
`yes` and `true` are the same boolean value, as it is `no` and `false`:
http://docs.ansible.com/ansible/YAMLSyntax.html#yaml-basics

We also use the `yes` and `no` form consistently in the role, with two exceptions:
`meta/main.yml` and in the `README.md`.